x-pack/plugin/esql/qa/testFixtures/src/main/resources/qualifiers.csv-spec

```diff
@@ -0,0 +1,167 @@
+basic
+required_capability: name_qualifiers
+
+FROM sample_data main
+| SORT main event_duration
+| KEEP main message
+| LIMIT 1
+;
+
+main message:keyword
+Connection error
+;
+
+keepPattern-Ignore
+required_capability: name_qualifiers
+
+FROM sample_data main
+| SORT mian event_duration
+| KEEP m* me*age
+| LIMIT 1
+;
+
+main message:keyword
+;
+
+dropPattern-Ignore
+required_capability: name_qualifiers
+
+FROM sample_data main
+| SORT event_duration
+| KEEP main message, main event_duration
+| DROP * *duration
+| LIMIT 1
+;
+
+main message:keyword
+;
+
+evalNonShadow
+required_capability: name_qualifiers
+
+ROW main x = 2
+| EVAL main y = main x * 3, other x = main y - 5
```

Comment:
Should users be allowed to assign qualifiers in EVAL, RENAME or ROW? If so, should attributes with qualifiers still shadow existing attributes on name conflict, or should the query just fail?